src/common/subsys_types.h

```diff
@@ -30,5 +32,25 @@ constexpr std::size_t ceph_subsys_get_num() {
   return static_cast<std::size_t>(ceph_subsys_max);
 }
 
+struct ceph_subsys_item_t {
+  const char* name;
+  uint8_t log_level;
+  uint8_t gather_level;
```

Comment:
Hmm, this might be a good measure to explicitly distinguish the hot path than the comment we have around `m_gather_levels`. Something targeting more the human's attention than machines. Fom x86's perspective I don't expect any difference. The open questions are: 1) the effect of *fast* types on ARMs and 2) the proliferation of `uint_fast8_t` across the codebase. This would be the first occurrence (not counting 3rd party code).